### PR TITLE
Remove PaymentAddress' languageCode

### LINKED ISSUE #608: Describe how to implement PaymentAddress.languageCode
Implementers are currently figuring out how to derive the `languageCode`'s value in an interoperable manner. 

During the CR phase of standardization, implementers will report back on implementation experience, possibly using [Unicode CLDR](http://cldr.unicode.org) to derive the `languageCode`. Please see comments of this GitHub issue for followup, or join the discussion.

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #608

The following tasks have been completed:

 * [x] Confirmed there are no ReSpec errors/warnings.
 * [x] [Added Web platform tests](https://github.com/web-platform-tests/wpt/pull/12684)
 * [x] [added MDN Docs](https://developer.mozilla.org/en-US/docs/Web/API/AddressErrors/languageCode)
 * [ ] added MDN [compat data](https://github.com/mdn/browser-compat-data)

Implementation commitment:

 * [x] [Safari](https://bugs.webkit.org/show_bug.cgi?id=189254)
 * [ ] [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=877521)
 * [x] [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1485881)
 * [ ] Edge (public signal)

Impact on Payment Handler spec?
None

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/pull/765.html" title="Last updated on Oct 8, 2018, 4:35 AM GMT (6b656bf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/765/007c785...6b656bf.html" title="Last updated on Oct 8, 2018, 4:35 AM GMT (6b656bf)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Updated MDN, and sent a PR to WTP to remove `languageCode`.

**Comment by marcoscaceres** [comment]
Chairs, we should decide on a timeframe for feedback on this from the community.

1 month CFC? 

I'm going to remove from Gecko in the mean time. Can always revert it later if we need it.

**Review by rsolomakhin — APPROVED** [review]
I'm adding a usage counter for languageCode in Chrome and will get results from stable in ~2 months.

**Comment by rsolomakhin** [comment]
The usage counter will inform our decision to remove the languageCode immediately or after some developer outreach.

**Comment by marcoscaceres** [comment]
@rsolomakhin 
> The usage counter will inform our decision to remove the languageCode immediately or after some developer outreach.

Great! Will you be notified automatically once the results are collected? I'll wait for your results before merging.

**Comment by rsolomakhin** [comment]
> Will you be notified automatically once the results are collected? 

I wish! I have a calendar reminder to check the data :-)

**Comment by marcoscaceres** [comment]
> I wish! I have a calendar reminder to check the data :-)

Good enough 👍

**Comment by marcoscaceres** [comment]
Removed from Firefox.

**Comment by aestes** [comment]
I've removed it from Safari as well.

**Comment by adrianhopebailie** [comment]
@marcoscaceres please tag me or @mountainhippo if you need anything from the chairs. Only saw this now.

@ianbjacobs is this covered by process? We never marked this as at risk previously

**Comment by ianbjacobs** [comment]
Hi all,

I had hoped we would hold off merging until after the CfC. That is why the draft CfC I circulated pointed to the pull request. My expectation was to merge after the CfC.

Ian

**Comment by marcoscaceres** [comment]
Yep, that’s the plan. It’s actually to wait for @rsolomakhin to gather data from Chrome before we make a final decision.

**Comment by marcoscaceres** [comment]
Call for Consensus:
https://lists.w3.org/Archives/Public/public-payments-wg/2018Sep/0005.html

**Comment by marcoscaceres** [comment]
@rsolomakhin, how’s that data looking? Any chance we will know by TPAC? We’ve removed this from Firefox and it’s also gone from Safari. Hopefully it’s all good news on the Chrome side.

**Comment by rsolomakhin** [comment]
Data shows we can remove from Chrome. I've sent out the [intent to deprecate](https://groups.google.com/a/chromium.org/d/msg/blink-dev/ma2J2RumrmM/nmhnTmB5CAAJ).

**Comment by rsolomakhin** [comment]
So, we will print a deprecation warning in developer console for a couple of months, then will remove it outright.

**Comment by marcoscaceres** [comment]
Awesome, thanks for confirming and doing the due diligence! Going to merge this then.

**Pull request merged**
